wordpress-mobile/WordPress-iOS
# Revisions: Add notice with undo button

Refs #10304

This adds a notice with undo button when loading a revision.

![revisions undo](https://user-images.githubusercontent.com/517257/49786271-10cff600-fcd8-11e8-9b21-1c75d27df895.gif)

**Note:** This currently also includes changes from #10633, and I'll wait until that is approved to merge.

To test:
- Load a revision from a posts's history
- Ensure a notice view appears
- Ensure that the undo button on the notice words

Update release notes:
